CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a check for schema and snapshot before attempting to start a local deployment. If either are missing then an error message will be displayed.
 - Added optional net relevancy check in replication prioritization. If enabled, an actor will only be replicated if IsNetRelevantFor is true for one of the connected client's views.
 - It is now possible to specify in Unreal which actors should not persist as entities in the Snapshot.
+- Add SpatialToggleMetricsDisplay console command.
```

Comment:
Probably worth mentioning that even with `bEnableMetricsDisplay` set to true, the visual part will be disabled before calling the command?

Reply:
sure i'll add that